### PR TITLE
Feat/tail

### PR DESCRIPTION
**Pull request by JoeyStewart**
What I did:

Various styling and 3 day weather modal implementation.
 
Did this break anything?
- [x] No
- [ ]  Yes
Type of change
- [x] New feature (non-breaking change which adds functionality)
- [x]  Styling
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.
Checklist:
- [ ]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [ ]  There are comments where clarification/ organization is needed
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing
What's next?

### CODE REVIEW

File: src/Carousel/Carousel.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    height: 300px;
+    object-fit: contain;
+    border-radius: 10px;
+}
+
+.carousel {
+    position: relative;
+}
+
+.carousel .button {
```
Comment (Jesuitman) [inline src/Carousel/Carousel.css:19]: var niceeeee

File: src/WeatherModal/WeatherModal.js [hunk 2]
```diff
@@ -8,24 +8,27 @@ function WeatherModal({ currentWeather, isOpen, onClose }) {
...
                                 alt="Weather Icon"
-                            />
+                                />
+                            <h3>Date: {day.date}</h3>
                             <p>{day.condition}</p>
-                            <p>Temperature: {day.temperature} °F</p>
-                            <p>Feels Like: {day.feels_like} °F</p>
-                            <p>Humidity: {day.humidity}%</p>
+                            <p>Max Temp: {day.max_temp} °F</p>
+                            <p>Min Temp: {day.min_temp} °F</p>
```
Comment (Jesuitman) [inline src/WeatherModal/WeatherModal.js:29]: amazing work on this

**Review by Jesuitman — APPROVED** [review]
This looks really good! way to go on getting the weather to work! it looks fantastic!

File: src/SelectedStateParks/SelectedStateParks.js [hunk 3]
```diff
@@ -5,16 +5,21 @@ import axios from "axios";
...
-function SelectedStateParks({ apiLink, parks, selectedState, setFoundPark, setParkCode }) {
+function SelectedStateParks({ apiLink, parks, selectedState, setFoundPark, setParkCode, weatherLink,  setWeather }) {
   const navigate = useNavigate();
   const fetchParkDetails = async (park_code) => {
     try {
       const response = await axios.get(`${apiLink}/${park_code}`);
       const selectedPark = response.data.data;
 
-      if (selectedPark) {
+      const weatherResponse = await axios.get(`${weatherLink.replace(':id', park_code)}`);
```
Comment (tenthwalker) [inline src/SelectedStateParks/SelectedStateParks.js:15]: I haven't seen the replace method before, cool to know that it exists!

**Pull request merged**